docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm

```diff
@@ -1,58 +1,475 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
 <html>
- <head>
-     <script type="text/javascript" src="../snippet.js"></script>
-     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-     <link href="../styles.css" rel="StyleSheet" type="text/css" />
-     <link href="../snippet.css" rel="StyleSheet" type="text/css" />
-     <title>Slider</title>
- </head>
- <body id="content">
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
+	<link href="../styles.css" rel="StyleSheet" type="text/css">
+	<link href="../snippet.css" rel="StyleSheet" type="text/css">
+	<title>Slider</title>
+</head>
+<body onload="prettyPrint()" id="content">
+
 
 <h1>Slider</h1>
 
-<p>The slider component shows on the screen a control that you can use to change values by dragging a handle on a horizontal scale.</p>
+<p>The slider component shows on the screen a control that you can use to change values by dragging a handle on a circular scale.</p>
+
+<h2>Table of Contents</h2>
+<ol class="toc">
+
+		<li><a href="#manual-constructor0.6728984923101962">Manual Constructor</a>
+
+		</li>
+
+		<li><a href="#html-examples0.5146947458852082">HTML Examples</a>
+						</li>
+
+		<li><a href="#options-list">Options</a>
+
+		</li>
+
+		<li><a href="#methods0.2250751326791942">Methods</a>
+
+		</li>
+
+</ol>
+
+<h2><a id="default-selectors0.42985791829414666"></a>Default Selectors</h2>
+
+<p>By default, all <code>&lt;input&gt;</code> elements with the <code>type=&quot;range&quot;</code> attribute and <code>data-type=&quot;range&quot;</code> and <code>data-role=&quot;slider&quot;</code> are displayed as Tizen Web UI sliders.</p>
+
+
+
+<h2><a id="manual-constructor0.6728984923101962"></a>Manual Constructor</h2>
+
+<p>To manually create a slider component, use the component constructor from the <code>tau</code> namespace</p>
 
-<p>To add a slider component to the application, use the following code:</p>
+<p>HTML code:</p>
 
-<video autoplay muted loop controls>
-	<source src="https://developer.tizen.org/sites/default/files/documentation/slider.mp4">
```

Comment:
This mp4 file is added only few days ago. Are you going to remove this video now?

Reply:
I think it should stay